src/qp/metrics/metrics.py

```diff
@@ -1,70 +1,143 @@
 """This module implements some performance metrics for distribution parameterization"""
```

Comment:
This diff looks really intimidating, but that's almost entirely due to me alphabetizing the functions in this module. The new code that needs to be reviewed in is the following functions:
`calculate_anderson_ksamp`
`calculate_cramer_von_mises`
`calculate_kolmogorov_smirnov `
`calculate_outlier_rate`
`_check_ensembles_are_same_size` and 
`_check_ensemble_is_not_nested`

Reply:
To avoid scary changes that aren't meaningful changes, I suggest first having a "prefactor" cleanup PR, where you do the alphabetizing and other semi-automated changes, so that more interesting changes are understandable (and cleanup PRs can be ignored in blame).

Reply:
Thanks, that's a great idea! I'll go ahead and back out the alphabetization here, just to make this file more tractable.

Reply:
I un-alphabetized the functions, so now the only changes are actual substantial code changes.